src/activities/ActivityUsageEntity.js

```diff
@@ -635,9 +635,11 @@ export class ActivityUsageEntity extends Entity {
 			return;
 		}
 
+		const scoreOutOf = this.scoreOutOf() ? this.scoreOutOf().toString() : undefined;
+
 		return !this._shouldCreateAssociationToNewGrade(scoreAndGrade) &&
 			(this._shouldCreateAssociationToExistingGrade(scoreAndGrade)
-			|| scoreAndGrade.scoreOutOf !== this.scoreOutOf().toString()
+			|| scoreAndGrade.scoreOutOf !== scoreOutOf
```

Comment:
i haven't tested it out, but single not equals would work I think as it will just ignore one of them being a string, so conversion isn't necessary. but that might not be super clear as to why it does that kind of comparison.
```
scoreAndGrade.scoreOutOf != this.scoreOutOf()
```

Reply:
i wouldn't really recommend this though..

Reply:
Yeah, not my code so don't want to mess with it too much. Don't want to blow up assignments stuff.